integration/erc20contract/build_erc20_contract.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+#
+# This script builds the guessing game from solidity to a go generated package
```

Comment:
@tudor-malene Same here - should there be a reference to the guessing game here?